### PR TITLE
Task GSGEODPT43-97

### PR DESCRIPTION
**Pull request by y-vynohradov**
Create a new application apps/datafeeder
Create a new called lib-editor in libs/editor

### CODE REVIEW

File: apps/datafeeder/src/app/app.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  })
+
+  it(`should have as title 'datafeeder'`, () => {
+    const fixture = TestBed.createComponent(AppComponent)
+    const app = fixture.componentInstance
+    expect(app.title).toEqual('datafeeder')
+  })
+
+  it('should render title', () => {
+    const fixture = TestBed.createComponent(AppComponent)
```
Comment (fgravin) [inline apps/datafeeder/src/app/app.component.spec.ts:26]: wonder how it passes as i see nothing in the template

File: angular.json [hunk 2]
```diff
@@ -508,6 +508,107 @@
...
+                {
+                  "type": "anyComponentStyle",
+                  "maximumWarning": "6kb",
+                  "maximumError": "10kb"
+                }
+              ]
+            }
+          }
+        },
+        "serve": {
```
Comment (fgravin) [inline angular.json:582]: miss test target
check test target from search app

**Review by fgravin — CHANGES_REQUESTED** [review]
Very good start !
Please add the test target for datafeeder application and fix your test.

Thanks

File: apps/datafeeder/src/app/app.component.spec.ts (hunk 1 above)
Comment (y-vynohradov) [reply apps/datafeeder/src/app/app.component.spec.ts]: It seems test was skipped because of miss test target in agular.json

**Review by fgravin — APPROVED** [review]
Thanks for the changes, excellent work

**Pull request merged**